pages/register.tsx

```diff
@@ -190,19 +194,11 @@ export default function Register({ allowedRegistrations }: RegisterPageProps) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <section className="pt-4 pl-4">
+        {/* TODO: update */}
```

Comment:
Will this need to be addressed before merge?

Reply:
forgot to remove this comment, will remove it